.github/workflows/swap-azure-web-app-slots.yml

```diff
@@ -29,6 +29,13 @@ on:
           The ID of the Azure Subscription the resources are under, which will be mapped to the subscription-id
           parameter when calling azure/login. You can look this up e.g. in the Azure Portal under any resource or the
           subscription itself.
+      TAG_DEPLOYMENT_TOKEN:
```

Comment:
I think more like `CODE_REPOSITORY_WRITE_TOKEN`. This sounds more like if we were to deploy tags; tagging is not really a "deployment", and we do a release too.